Sources/FluentKit/Properties/Children+EagerLoading.swift

```diff
@@ -0,0 +1,94 @@
+private protocol SubqueryLoader: EagerLoadRequest {
```

Comment:
Do we need a protocol here? I think it could be done as a concrete type like before.